### PR TITLE
chore: update e2e test docs

### PR DESCRIPTION
**Pull request by joyceyan**
## Reason for Change

i had trouble running e2e tests locally

## Changes

updates the e2e test docs based on some feedback from @tihuan and @kaloster !

## Testing steps

n/a

## Checklist 🛎️

- [ ] Add product, design, and eng as reviewers for rdev review

- [ ] For UI changes, add screenshots/videos, so the reviewers know what you expect them to see

- [ ] For UI changes, add e2e tests to prevent regressions

### CODE REVIEW

**Review by tihuan — APPROVED** [review]
This is super helpful! Thanks so much for adding these, @joyceyan 🌟 🏆 !!

File: frontend/tests/README.md [hunk 1]
```diff
@@ -228,6 +228,47 @@ Before running any tests, start with [mise en place](../README.md#mise-en-place)
...
+```
+  ✘  2 [setup] › tests/common/playwright.global.setup.ts:10:7 › global setup › login (13.3s)
+🔐🪵 Logging in...
+Error: Region is missing
+```
+
+You can resolve this by updating your AWS config file to include the following snippet. On a Linux/Mac, this should be in `~/.aws/config`.
+```
+[default]
+role_arn       = arn:aws:iam::699936264352:role/poweruser
```
Comment (tihuan) [inline frontend/tests/README.md:267]: Not sure if we should redact this for security? I see Github does that in the lint error output 😆 

```
[default]
 role_arn       = arn:aws:iam::***:role/poweruser // <--- THIS
Error: Process completed with exit code 1.
```

Comment (joyceyan) [reply frontend/tests/README.md]: ah good call.. will do!

**Review by tihuan — APPROVED** [review]
Thanks for updating the PR 🙏 !

**Pull request merged**